signalfx-org-metrics/docs/sf.org.numDimensionValuesCreated.md

```diff
@@ -0,0 +1,10 @@
+---
+title: sf.org.numDimensionValuesCreated
+brief: Number of new dimension values that were created
+metric_type: gauge
+---
+### sf.org.numDimensionValuesCreated
+
+Count of each new dimension name values that were created in the past 5 minutes in this organization
```

Comment:
Wording "name value" is confusing. Since there is another metric, numDimensionsCreated, that counts new dimensions created, this must be number of new values for an existing dimension (e.g. the metric has region:east) where region is an existing dimension but east is a new value for that dimension. Correct?

Or maybe I'm not really clear on what this metric means....